spec/views/collections/_show_descriptions.html.erb_spec.rb

```diff
@@ -13,6 +13,7 @@
     end
     before do
       assign(:presenter, presenter)
+      allow(Sufia::CollectionSizeService).to receive(:run).and_return(123456678)
```

Comment:
Ah, I now see the value up above in "bytes".  Perhaps put this number into a let statement and use that variable (e.g., "bytes") in both places?

`let(:bytes) { 123456678 }`